src/main/java/oshi/software/os/linux/LinuxFileSystem.java

```diff
@@ -76,9 +101,8 @@
             // FileStore toString starts with path, then a space, then name in
             // parentheses e.g., "/ (/dev/sda1)" and "/proc (proc)"
             String path = store.toString().replace(" (" + store.name() + ")", "");
-            // Exclude special directories
-            if (path.startsWith("/proc") || path.startsWith("/sys") || path.startsWith("/run")
-                    || path.startsWith("/dev"))
+            // Exclude pseudo file systems
+            if (Arrays.asList(this.pseudofs).contains(store.name()))

```

Comment:
change of the approach is necessary because in some distributions (e.g. Fedora) external storage (such as USB key) are mounted in `/run/media/$user/mountpoint`